Rx.NET/Source/src/System.Reactive/System.Reactive.Wasm.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageId>Rx.Uno.Wasm</PackageId>
+    <Version>1.0.0-alpha</Version>
+    <Authors>nventive, shimmy</Authors>
+    <Product>Rx.Uno.Wasm</Product>
+    <Description>Uno Wasm implementation for System.Reactive</Description>
```

Comment:
If updated to test for Threading availability, if becomes a generic thread-less implementation which is not specific to Uno.

Reply:
This was resolved.